inventory/any.cf

```diff
@@ -685,6 +688,7 @@ bundle agent cfe_autorun_inventory_aws_ec2_metadata_cache
     imdsv2_result_valid::
       "$(cache)"
         content => "$(cfe_autorun_inventory_aws_ec2_metadata_data.imdsv2_cmd_result)";
+@endif
 }
 
 bundle agent cfe_aws_ec2_metadata_from_cache
```

Comment:
@nickanderson shouldn't this one also be disabled?

Reply:
It didn't have any incompatible things.

Reply:
so because this bundle depends on data from a potentially disabled/empty bundle it just won't "do" anything.

seems pretty OK to me.